test/data_base.cpp

```diff
@@ -222,8 +222,7 @@ static chain_state::data data_for_chain_state()
 
 static void set_state(block& block)
 {
-    auto state = std::make_shared<chain_state>(
-        chain_state{ data_for_chain_state(), {}, 0, 0, {} });
+    const auto state = std::make_shared<chain_state>(chain_state{ data_for_chain_state(), {}, 0, 0, {} });
```

Comment:
Test utilities are considered source not considered tests, which means they should be tested and confirm to source style (80 char).